inscriptions/ISic000526.xml

```diff
@@ -190,11 +195,15 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/><supplied reason="lost"><expan><abbr>d</abbr><ex>is</ex></expan></supplied> <supplied reason="lost"><g ref="#interpunct">·</g></supplied> <expan><abbr>m</abbr><ex>anibus</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>s</abbr><ex>acrum</ex></expan> <g ref="#interpunct">·</g>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/>us <g ref="#interpunct">·</g> <name>Saturio</name>
```

Comment:
you could do gap followed by name with seg and part = F here, as above for the '-us'